tests/python-pytest/onnx/test_onnxruntime_cv.py

```diff
@@ -240,14 +238,20 @@ def obj_detection_test_images(tmpdir_factory):
     'faster_rcnn_resnet101_v1d_coco',
     'yolo3_darknet53_coco',
     'yolo3_mobilenet1.0_coco',
+    'mask_rcnn_resnet18_v1b_coco',
```

Comment:
By adding the mask_rcnn under obj_detection tests, we are not testing the segmentation part?

Reply:
I think the mask is a little hard to test numerically since they can be different across mxnet and onnxruntime